content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,350 @@
+
+After configuring the company in the database settings, navigating to :menuselection:`Contacts` and
+search for your company to verify the following:
+
+- Company type is set to :guilabel:`Company`.
+- Verify the :guilabel:`Identification Number` :guilabel:`Type` is :guilabel:`RUT / RUC`.
+
+.. _l10n_uy/uruware-account:
+
+Set up Uruware account
+----------------------
+
+To set up a Uruware account, first be sure to have a valid Odoo subscription. Then follow the
+proceeding steps.
+
+Locate the Uruware credentials settings by navigating to the :menuselection:`Accounting -->
+Configurations --> Settings --> Localización Uruguaya`. Then, click on :guilabel:`Create Uruware
+Account`.
+
+Upon doing so, an email is sent to the address associated with your Odoo subscription with the
+username and password to enter Uruware's portal and setup your account.
+
+.. tip::
+   Your :guilabel:`Tax ID` needs to be setup to be able to create your account.
+
+.. note::
+   This action will create an account with Uruware with the following information:
+
+   - Legal name (razón social)
+   - RUT from the company
+   - Administrator email
+   - User name and last name
+   - Odoo database link
+
+   To ensure your account is created correctly, please add any missing information from above.
+
+Once the account is created, a confirmation email is sent to the email used to create the account,
+containing the account credentials to configure the account directly in the Uruware portal:
+
+- Testing portal: https://odootest.ucfe.com.uy/Gestion/
+- Production portal: https://prod6109.ucfe.com.uy/Gestion/
+
+Use the account credentials in the email to log in to the portal.
+
+.. important::
+   Be sure to configure two accounts, one for testing and one for production. The email address
+   can be changed in Uruware and the same can be used in both environments.
+
+Electronic invoice data
+-----------------------
+
+To configure the electronic invoice data, an environment and credentials need to be configured. To
+do so, navigate to :menuselection:`Accounting --> Configurations --> Uruguayan Localization`.
+
+First, select the :guilabel:`UCFE Web Services` environment:
+
+- :guilabel:`Production`: for production databases. In this mode, electronic documents are sent to
+  |DGI| through Uruware for their validation.
+- :guilabel:`Testing`: for test databases. In this mode, the direct connection flows can be tested,
+  with the files sent to the |DGI| testing environment through Uruware.
+- :guilabel:`Demo`: files are created and accepted automatically in demo mode but are **not** sent
+  to the |DGI|. For this reason, rejection errors will not appear in this mode. Every internal
+  validation can be tested in demo mode. Avoid selecting this option in a production database.
+
+Then, enter the :guilabel:`Uruware Data`:
+
+- :guilabel:`Uruware WS Password`
+- :guilabel:`Commerce Code`
+- :guilabel:`Terminal Code`
+
+.. image:: uruguay/electronic-invoice-data.png
```

Comment:
Hello @samueljlieber 

One quick change since I told you the change in the text but didn't see the image. Can you change the above image to this one (will send you the file through discord):

![Screenshot 2024-09-05 at 2 13 22 PM](https://github.com/user-attachments/assets/35673177-ed72-493f-a05f-ba299a038c59)

Let me know if anything!

Thanks!